internal/backends/winit/winitwindowadapter.rs

```diff
@@ -539,9 +535,7 @@ impl WindowAdapter for WinitWindowAdapter {
                     winit_window.set_fullscreen(Some(winit::window::Fullscreen::Borderless(None)));
                 }
             } else {
-                if winit_window.fullscreen().is_some() {
-                    winit_window.set_fullscreen(None);
-                }
+                winit_window.set_fullscreen(None);
```

Comment:
I think this check is still needed, isn't it? See 8caa5acd304b1020d3fa6422170d107dc20fa3b8